src/control-plane/backend/lambda/api/service/attribution.ts

```diff
@@ -110,8 +110,8 @@ export class AttributionAnalysisService {
       projectedColumns: [
         'Trigger Count',
         'Touch Point Name',
-        'Number of Attributed Triggers',
-        'Contribution',
+        'Number of Triggers with Conversion',
```

Comment:
Where is the column name from? Should we make them constants?

Reply:
Only show in table when explore result.